deploy/olm-catalog/argocd-operator/0.19.0/argoproj.io_argocds.yaml

```
@@ -19073,6 +19118,11 @@ spec:
                     description: Custom labels to pods deployed by the operator
                     type: object
                   remote:
+                  priorityClassName:
+                    description: |-
+                      PriorityClassName is the name of the PriorityClass to assign to the Redis pod.
+                      The PriorityClass must already exist in the cluster.
+                    type: string
                     description: Remote specifies the remote URL of the Redis container.
```

YAMLlint (1.37.1)

[error] duplication of key "description" in mapping

(key-duplicates)